packages/sdk/src/utils.ts

```diff
@@ -4,6 +4,9 @@ import type Multicall2 from "./multicall2";
 import type { Contract } from "ethers";
 import zip from "lodash/zip";
 
+// re-use this class, its compatible with FE and BE
+export { BlockFinder } from "@uma/financial-templates-lib/dist/price-feed/utils";
```

Comment:
How does this work? Will the frontend be able to access this package if it isn't a runtime dependency of the sdk?

Also, accessing files within packages like this isn't portable (I don't think). I've seen problems when trying to do this in a frontend because the filesystem doesn't exist like it does in node.

Reply:
im going to try it now before i merge this, will let you know. I think it might work though because the sdk gets built indepdendently before importing into FE.

Reply:
yea it doesnt work :( i will have to copy it into the sdk i think